hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -765,6 +765,14 @@ public class HoodieWriteConfig extends HoodieConfig {
       .withDocumentation("Avro schema of the partial updates. This is automatically set by the "
           + "Hudi write client and user is not expected to manually change the value.");
 
+  public static final ConfigProperty<Boolean> WRITE_IGNORE_FAILED = ConfigProperty
```

Comment:
@xuzifu666 : This is handled by the Hudi layer above core part. For example : In HoodieStreamer, we have this flag --commit-on-errors. Also Spark writers handles this way : https://github.com/apache/hudi/blob/f39cb1d2e3e5eece1542252316f4135cec83299b/hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala#L956 . Adding this config would make the other configs and related code redundant. Can you look into exploring those sections of code more and handling this at the hudi-spark layer ?

Reply:
Ok，I would try to do the change@bvaradar

Reply:
Makes sense, Flink also has the duplicate logic.

Reply:
@danny0405 also need to be changed or keep the pr current logic to it？

Reply:
@bvaradar OK，if so I would handle this in hudi-spark layer，that seems handle this in a better way.

Reply:
@bvaradar @danny0405 Had fix it in spark layer，and leave the config for it，PTAL